### DISCUSSION TITLE
How to export deconvoluted plot from UniChrom sliding window analysis

### DISCUSSION THREAD

**Discussion by garganandrea** [michaelmarty/UniDec#92] [Q&A]
Hi all,
I deconvoluted an LC-MS analysis using the sliding window tool and I extracted a deconvoluted mass spectra that I would like to export as text file.
Using control+U I can get some of the deconvoluted traces but not the extracted/ plotted sum trace (here in black).

![image](https://github.com/michaelmarty/UniDec/assets/40744633/7fa3007f-54bc-4cf7-8cfe-944a6ae73f53)

 Any trick to do this?

I want to use it to overlay the results of different analysis.

Thanks for the help!

Andrea

**Comment by michaelmarty** [comment]
Hi Andrea,
We could definitely add a text export if needed, but the info is all in already the HDF5 if you know where to look and how to get to it. 

You can either access it programmatically (h5py is a great python library and there are few tools in the UniDec hdf5_tools.py file) or with HDF View (https://www.hdfgroup.org/downloads/hdfview/). 

If you open it up in HDF View, under the ms_dataset object, there should be a mass_axis object with the x-axis info and a mass_sum object with the y-axis intensities. You can copy/paste out of HDF View if needed.

Good luck!
MTM
